apps/namada-interface/src/slices/notifications.ts

```diff
@@ -51,6 +60,22 @@ export const NotificationsSlice = createSlice({
       state.toasts = newToasts;
     },
   },
+  extraReducers: (builder) => {
+    builder.addMatcher(isAsyncThunkAction, (state, action) => {
+      const [actionType] =
+        action.type.match(/^.*(?=\/(pending|fulfilled|rejected)$)/) || [];
```

Comment:
nit: even though it is obvious what we are after here, it might be pleasant to have a few practical examples. Alternatively, we could have a follow up task to put this to it's own folder and have a small test for it.
```suggestion
        // match example "actionIdentifier/fulfilled" => ["actionIdentifier"]
        // match example "stakingAndGovernance/FETCH_VALIDATORS/pending" => ["stakingAndGovernance/FETCH_VALIDATORS"]
        action.type.match(/^.*(?=\/(pending|fulfilled|rejected)$)/) || [];
```

Reply:
Added tests